jobs/speak-clearly-llc-python-telephony-developer.html

```diff
@@ -10,17 +10,17 @@
     ghangouts: jsalsman
 created: !!timestamp 2019-07-09
 tags: 
-  - Python	
-  - Twilio	
-  - Flask	
-  - Javascript 
-  - WebRTC	
-  - C/C++	
-  - PostgreSQL	
-  - Git	
-  - GitLab CI	
-  - Docker	
-  - Cross-validation	
+  - Python
+  - Twilio
+  - Flask
+  - Javascript
+  - WebRTC
+  - C/C++
+  - PostgreSQL
+  - Git
+  - GitLab CI
+  - Docker
+  - Cross-validation
```

Comment:
I'm not sure that cross-validation is a useful tag here. Are there any synonyms for this you might use?